### PR TITLE
CONSOLE-2437: i18n common components: dropdowns

### PR DESCRIPTION
**Pull request by rebeccaalpert**
Addressed shared dropdown, kebab, and project selector.

Related to https://issues.redhat.com/browse/CONSOLE-2437.

### CODE REVIEW

File: frontend/public/components/namespace.jsx
Comment (spadgett) [inline frontend/public/components/namespace.jsx:1001]: If we're translating projects and namespaces in the nav, we probably want to translate it here, too. We should know if it's one or the other and use different messages since those are the only two possibilities.

File: frontend/public/components/utils/dropdown.jsx
Comment (spadgett) [inline frontend/public/components/utils/dropdown.jsx:729]:
```suggestion
          [initContainer.name]: t('dropdown~Init containers'),
```

**Comment by rebeccaalpert** [comment]
This is ready for review again @spadgett.

File: frontend/public/components/namespace.jsx
Comment (spadgett) [inline frontend/public/components/namespace.jsx:1003]: We probably want to change the case here

```suggestion
      model.label === 'Project' ? t('dropdown~All Projects') : t('dropdown~All Namespaces');
```

File: frontend/public/components/namespace.jsx
Comment (spadgett) [inline frontend/public/components/namespace.jsx:1066]:
```suggestion
              ? t('dropdown~Select Project...')
              : t('dropdown~Select Namespace...')
```

**Review by spadgett — COMMENTED** [review]
lgtm, just a couple small comments

**Comment by spadgett** [comment]
/assign @yapei @ahardin-rh @sferich888 
for aprovals

**Comment by rebeccaalpert** [comment]
String updates all set @spadgett.

**Review by spadgett — APPROVED** [review]
/lgtm
/hold for approvals

**Comment by rebeccaalpert** [comment]
Had to rebase `frontend/public/locales/en/details-page.json` after the details page one merged. Lost the lgtm @spadgett.

**Review by spadgett — APPROVED** [review]
/lgtm

**Pull request merged**
